codebase/superdarn/src.lib/tk/fitacfex2.1.0/src/fitacfex2.c

```diff
@@ -614,10 +604,25 @@ void fitacfex2(struct RadarParm *prm, struct RawData *raw,
 
           }
           phi0 = calc_phi0(good_lags,xcf_phases, w_guess, goodcnt)*PI/180.;
+          //TODO calculate phi0_error
           fit->xrng[R].phi0 = phi0;
-          fit->elv[R].normal = elevation(&fblk->prm,phi0);
-          fit->elv[R].high = elevation(&fblk->prm,phi0);
-          fit->elv[R].low = elevation(&fblk->prm,phi0);
+          
+          if (fblk->prm.old_elev) {
+          /* use old elevation angle routines 
+             NB: elev_goose() has never been used in fitacfex2 */
+             fit->elv[R].normal = elevation(&fblk->prm,phi0);
+           
+          /* TODO This is just calculating the elevation angle again. 
+             Need to determine phi0_error.*/
+             fit->elv[R].high = elevation(&fblk->prm,phi0);
+             fit->elv[R].low = elevation(&fblk->prm,phi0);
+          }
+          else {
+            /* use the Shepherd elevation angle routine */
+            fit->elv[R].normal = elevation_v2(&fblk->prm,phi0);
+            fit->elv[R].high = 0.; // no error calculation since we don't have phi0_error
+            fit->elv[R].low = 0.;  // no error calculation since we don't have phi0_error
```

Comment:
Any reason to calculate elv high and low for the old elevation angle algorithm but not the Shepherd / v2 algorithm?  Could they both be set to zero for now?